bluebrain/repo-patches/packages/neuron/package.py

```diff
@@ -84,7 +79,6 @@ class Neuron(CMakePackage):
 
     depends_on("mpi",         when="+mpi")
     depends_on("py-mpi4py",   when="+mpi+python+tests")
-    depends_on("ncurses",     when="~cross-compile")
```

Comment:
Cross-compile is off by default, so shouldn't this be a dependency?

Reply:
Well spotted; fixed.